load-balancing/elb-v2/common_functions.sh

```diff
@@ -374,15 +381,21 @@ wait_for_state() {
         return 1
     fi
 
-    msg "Checking $WAITER_ATTEMPTS times, every $waiter_interval seconds, for instance $instance_id to be in state $state_name"
+    # Check if a custom waiter_attempts was passed into the function
+    # and override the attemps if true
+    if [ -z "$waiter_attempts" ]; then
```

Comment:
I believe that it's actually supposed to be `${waiter_attempts+x}`. However, in this case, being an empty string or null are essentially the same, so I think `"$waiter_attempts"` is the right way.